src/Components/calculator.js

```diff
@@ -1,6 +1,7 @@
 import './calculator.css';
 import { useState } from 'react';
 import calculators from '../Logic/calculators';
+import Quote from './Quote';
```

Comment:
- [ ] Great job implementing the quotes component, however, the requirement is that the `Quotes` component should be imported into the main component in this case the `App.js`, kindly fix this
![image](https://user-images.githubusercontent.com/81932478/236413068-33022270-4c3f-4f35-8264-b13a51100712.png)
[Requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_fetch_data_from_api.md)

Reply:
This change has been done well